packages/core/src/graphql/geo-locations/merchants/geo-location-merchants.resolver.ts

```diff
@@ -18,7 +18,7 @@ export class GeoLocationMerchantsResolver {
 		let merchants = await this.geoLocationsWarehousesService.getMerchants(
 			geoLocation,
 			IN_STORE_DISTANCE,
-			{ fullProducts: false, activeOnly: true }
+			{ fullProducts: false, activeOnly: true, storeMode: true }
```

Comment:
@sunk0 why `storeMode`??? We talked about it today.........

Reply:
@evereq ah didnt saw that

Reply:
@evereq fixing